components/sbm-core/src/main/java/org/springframework/sbm/build/migration/recipe/AddMavenPlugin.java

```diff
@@ -108,7 +110,7 @@ private String createPluginTagString() {
 		sb.append("</artifactId>\n");
```

Comment:
Not your code, but what about moving the rendering `createPluginTagString` into `Plugin`?
E.g. `plugin.render()`